packages/next/server/config.ts

```diff
@@ -320,6 +320,12 @@ function assignDefaults(userConfig: { [key: string]: any }) {
       )
     }
 
+    if (i18n.locales.length > 100) {
+      throw new Error(
+        `Received ${i18n.locales.length} i18n.locales items which exceeds the max of 100, please reduce the number of items to continue.\nSee more info here: https://nextjs.org/docs/messages/invalid-i18n-config`
```

Comment:
This error page doesn't mention the limit

Reply:
Good point, added it in that doc as well